features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/Resource.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2017, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
2017 -> 2018

Reply:
We should remove the year right?

Reply:
I see in other js files the year is 2018.  Any reason to remove?

Reply:
AFAIK that's the standard now. The reason should be because the year becomes obsolete since the copyright applies to not just the year specified but forever since.

Reply:
Right. That makes sense. So it's not only for JS files but for all the files.